packages/drivers/odsp-driver/src/odspDocumentStorageManager.ts

```diff
@@ -466,17 +466,64 @@ export class OdspDocumentStorageService implements IDocumentStorageService {
         // snapshot code path. We should leverage the fact that these deltas are returned to speed up the deltas fetch.
         const { headers, url } = getUrlAndHeadersWithAuth(`${this.snapshotUrl}/trees/latest${snapshotOptions}`, storageToken);
 
+        let isOptionsCall = false;
+
+        if (headers.Authorization) {
```

Comment:
worth adding a small comment explaining this for future reference.

Reply:
@ram-ramasubramanian I think if any header is present then it would result in options call. So I think you should check the length of keys in headers object instead of jusst Auth.

Reply:
Also you can check this in localhost by using : npm run start:spo-df
It will then uise this driver.

Reply:
I'm running tests using npm run start:spo-df only. I modified the check to if (Object.keys(headers).length), pl. let me know if this works fine, during my tests, the lenght comes as 1 when Authorization token is there.